libs/dh/market-participant/actors/feature-organizations/src/lib/drawer/tabs/dh-organization-history.component.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, computed, effect, input } from '@angular/core';
+
+import { TranslocoDirective } from '@ngneat/transloco';
+
+import { WATT_CARD } from '@energinet-datahub/watt/card';
+import { WattDatePipe } from '@energinet-datahub/watt/date';
+import { VaterStackComponent } from '@energinet-datahub/watt/vater';
+import { WattSpinnerComponent } from '@energinet-datahub/watt/spinner';
+import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
+import { WATT_TABLE, WattTableColumnDef, WattTableDataSource } from '@energinet-datahub/watt/table';
+
+import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+
+import {
+  GetAuditLogByOrganizationIdDocument,
+  OrganizationAuditedChangeAuditLogDto,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+
+@Component({
+  selector: 'dh-organization-history',
+  standalone: true,
+  template: ` <ng-container
+    *transloco="let t; read: 'marketParticipant.organizationsOverview.drawer'"
+  >
+    <vater-stack [fill]="'horizontal'" [align]="'center'">
+      @if (isLoading()) {
+        <watt-spinner />
+      }
+    </vater-stack>
```

Comment:
suggestion: it's just a copy/paste from before but it looks like the `vater-stack` element will still be in the DOM and be empty when `isLoading` is `false`. Let's move the `if` one level up.

```suggestion
    @if (isLoading()) {
      <vater-stack fill="horizontal" align="center">
        <watt-spinner />
      </vater-stack>
    }
```